.github/workflows/push_encfs_image.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           else  
             branch_name=${{ github.head_ref }}
           fi
+          branch_name=$(echo $branch_name | sed 's/[^a-zA-Z0-9]/-/g')
```

Comment:
Do we need check for image name length just in case?

Reply:
https://github.com/microsoft/confidential-sidecar-containers/pull/32/commits/22a887b0d36eab3af296f64f433dbf63637ccb25